web/packages/teleterm/src/ui/ModalsHost/modals/ReAuthenticate/ReAuthenticate.tsx

```diff
@@ -43,39 +43,34 @@ import { FieldSelect } from 'shared/components/FieldSelect';
 
 import { Option } from 'shared/components/Select';
 
-import { assertUnreachable } from 'shared/utils/assertUnreachable';
-
 import { useAppContext } from 'teleterm/ui/appContextProvider';
 import { LinearProgress } from 'teleterm/ui/components/LinearProgress';
 import svgHardwareKey from 'teleterm/ui/ClusterConnect/ClusterLogin/FormLogin/PromptPasswordless/hardware.svg';
-import { useLogger } from 'teleterm/ui/hooks/useLogger';
 import { routing } from 'teleterm/ui/uri';
+import PromptSsoStatus from 'teleterm/ui/ClusterConnect/ClusterLogin/FormLogin/PromptSsoStatus';
```

Comment:
I recently modified this component in #48143, but I just cherry-picked that merge commit and it looks even better after those changes when embedded within `ReAuthenticate` ([I removed extra padding from `PromptSsoStatus`](https://github.com/gravitational/teleport/pull/48143/files#diff-f9a474383edee28fd2f7b80c3d258fba93c46ec0558aab6185c41f73d766ca27)).